larray/inout/xw_excel.py

```diff
@@ -31,6 +31,42 @@
     global_app = None
 
 
+    def _create_instance(app_id):
+        def _create_instance(app_id):
```

Comment:
having the two different _create_instance functions with the same name is a recipe for disaster IMO